vpn_slice/__main__.py

```diff
@@ -365,6 +373,16 @@ def parse_args_and_env(args=None, environ=os.environ):
     env = parse_env(environ)
     return p, args, env
 
+def parse_routes_from_list(args, routes):
+    for x in routes:
```

Comment:
Parameter `x` need to parse by `net_or_host_param()` before checking.
Otherwise it will not handle the subnet and alias cases.

Reply:
please check https://github.com/dlenski/vpn-slice/pull/42/commits/2f0b4643b11fc067c1dd9e434370513103c346d5